pkg/controller/aws.go

```diff
@@ -175,7 +176,9 @@ func Setup(mgr ctrl.Manager, l logging.Logger, rl workqueue.RateLimiter, poll ti
 		distribution.SetupDistribution,
 		resolverendpoint.SetupResolverEndpoint,
 		resolverrule.SetupResolverRule,
+		broker.SetupBroker,
 		kafkacluster.SetupCluster,
+		broker.SetupBroker,
```

Comment:
```suggestion
		kafkacluster.SetupCluster,
		broker.SetupBroker,
```

We only need one SetupBroker.